frame/staking/src/tests.rs

```diff
@@ -5233,26 +5233,12 @@ fn proportional_ledger_slash_works() {
 	ledger.active = unit;
 	ledger.total = unit * 4 + value;
 	// When
-	assert_eq!(ledger.slash(slash, 0, 0), slash - 43);
+	assert_eq!(ledger.slash(slash, 0, 2), slash);
```

Comment:
Please add this test case and make sure it is reasonable as well: https://github.com/paritytech/substrate/commit/bdd139989cb4dfa8cc6cc6b5ee470a711df32d25

Ideally, existing tests should not be changed at all.

Reply:
Not sure why you changed this test?

Reply:
Oh sorry, I thought this case was subject to the round down problem and changed it to meet the case `not affected chunk should not be slashed` (like https://github.com/paritytech/substrate/commit/bdd139989cb4dfa8cc6cc6b5ee470a711df32d25), should not changing it and add a new case instead.